src/store/Business/mutations.ts

```diff
@@ -1,33 +1,57 @@
-// import Vue from 'vue'
-import { BusinessIF } from '@/interfaces'
+import { EntityState } from '@/enums'
+import { ApiBusinessIF, BusinessStateIF } from '@/interfaces'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
 
 export default {
-  // *** TODO: discuss this with Jonathan
-  setBusinessInfo (state: BusinessIF, val: BusinessIF) {
-    // state = val // doesn't work
+  setAdminFreeze (state: BusinessStateIF, val: boolean) {
+    state.businessInfo.adminFreeze = val
+  },
+
+  setBusinessInfo (state: BusinessStateIF, val: ApiBusinessIF) {
+    state.businessInfo = val
```

Comment:
The business info object is now a store property, instead of being "the whole store" as before.

Reply:
When the business info object is read from the API, this ^^ mutation is used.

The other mutations in here are primarily for test purposes (ie, where we want to set just one of the business info properties).